internal/incident/incident.go

```diff
@@ -173,13 +182,22 @@ func (i *Incident) ProcessEvent(ctx context.Context, ev *event.Event) error {
 		}
 	case event.TypeAcknowledgementSet:
 		if err := i.processAcknowledgementEvent(ctx, tx, ev); err != nil {
+			if errors.Is(err, errSuperfluousAckEvent) {
+				// That ack error type indicates that the acknowledgement author was already a manager, thus
+				// we can safely ignore that event and return without even committing the DB transaction.
+				return nil
+			}
+
 			return err
 		}
 	}
 
-	notifications, err := i.addPendingNotifications(ctx, tx, ev, i.getRecipientsChannel(ev.Time))
-	if err != nil {
-		return err
+	var notifications []*NotificationEntry
+	if ev.Type != event.TypeDowntimeStart && ev.Type != event.TypeAcknowledgementSet && ev.Type != event.TypeFlappingStart {
```

Comment:
This intentionally suppresses sending the event suppressing further events, e.g., the Downtime Start. However, the Downtime End notification will then be sent. I would expect to be notified that further notifications are stopped as Icinga 2 internal notification for starting downtimes.